shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -184,13 +276,18 @@ - (void)dealloc {
   [_markedTextRange release];
   [_selectedTextRange release];
   [_tokenizer release];
+  [_autofillId release];
   [super dealloc];
 }
 
 - (void)setTextInputClient:(int)client {
   _textInputClient = client;
 }
 
+- (void)setAutofillId:(NSString*)autofillId {
+  _autofillId = [autofillId copy];
```

Comment:
This doesn't release the old value.  Instead of writing this you can just do`@property(nonatomic, copy) NSString* autofillId` to have one automatically generated.  Put it in the .mm file to make it private.

Reply:
Thank you for the review! I have to put the interface in the header file in order to be able to import that in a unit test (at least for now). Does adding `autorelease` after `copy` work here too?

Reply:
Oh your pull request is already merged. I guess that means I can move the interface back to the .mm file?

Reply:
Yep, autorelease can be used on any object.